content/20.treatments.md

```diff
@@ -55,28 +55,30 @@ Also, RA patients with chronic hepatitis B (HB) infection showed high risk of HB
 These last findings highlight the need to search for a balance between impairing a harmful immune response, such as the one generated by the cytokine storm, and preventing the worsening of the clinical picture of the patients by potential new viral infections.
 This aspect is probably crucial to be investigated further in the trials that are about to start.
 Perhaps, the TCZ treatment would best suit patients with severely compromised lungs due to the Covid-19 infection and are therefore at greater risk of death, in order to stop the uncontrolled immune response before it’s too late.
-
+<!--
 ##### Summary
 
 Summarize the state of the symptom management approach.
+-->
 
-### Small Molecule Drugs for Targeting SARS-CoV-2
+### 4.2 Small Molecule Drugs for Targeting SARS-CoV-2
 
 The replication cycle of a virus within an epithelial host cell includes 6 basic steps which can be shortly summarised as follow: i) attachment of the virus to the host cell; ii) penetration by endocytosis; iii) uncoating, classically defined as the release of viral contents into the host cell; iv) biosynthesis, during which the viral genetic material enters the nucleus where it gets replicated; v) assembly, where viral proteins are translated and new viral particles are assembled; vi) release, when the new viruses are released into the extracellular environment [@isbn:978-1405136457].
 Antiviral drugs do not kill the virus, rather they inhibit its amplification by impairing one of these steps.
 Nowadays, many of these drugs act during the biosynthesis step in order to inhibit the virus’ genetic material replication.
 Importantly, SARS-CoV-2 is an RNA virus.
 Differently from a DNA virus, which can use the host enzymes to propagate itself, RNA viruses depends on their own polymerase, the RNA-dependent RNA polymerase (RdRP), in order to be replicated [@doi:10.1007/978-1-4939-2438-7; @doi:10.1002/jmv.25761].
 
-#### Nucleotide Analogs
-
+<!--#### 4.2.a Nucleotide Analogs
+Removing this header for now, if we add additional nucleotide analogs, we can put it back
```

Comment:
I think both of the drugs we have in this section are nucleotide analogs or precursors.

Reply:
Oh no, I got lost while trying to make it easier to follow! Hopefully fixed now.